test/python/transpiler/test_calibrationbuilder.py

```diff
@@ -62,13 +63,17 @@ def test_rzx_calibration_builder_duration(self, theta: float):
         amp = 1.0
         pulse = GaussianSquare(duration=duration, amp=amp, sigma=sigma, width=width)
         instruction = Play(pulse, ControlChannel(1))
-        scaled = RZXCalibrationBuilder.rescale_cr_inst(instruction, theta, sample_mult=sample_mult)
+        with builder.build():
+            # this is builder macro to play stretched pulse. need builder context.
+            test_duration = RZXCalibrationBuilder.rescale_cr_inst(
+                instruction=instruction, theta=theta, sample_mult=sample_mult
+            )
```

Comment:
Is it now required to run the pass in a builder context manager? If so that's the Z in my comment on the release notes above.